test/test-v8debugapi.ts

```diff
@@ -500,6 +500,29 @@ describe('v8debugapi', function() {
 
     });
 
+    it('should resolve correct breakpoint line number', function(done) {
+      // clone a clean breakpointInFoo
+      // TODO: Have this actually implement Breakpoint
```

Comment:
Is it possible to address these two TODOs in this PR?

Reply:
copied from other part. not TODO needed here. I will remove these TODOs.